### PR TITLE
Added svelte path

### PR DESCRIPTION
**Pull request by shaji-stark**
Found this repo interesting so added a framework that I found smooth to start off as a beginner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -502,6 +503,16 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Svelte
+
+- [Svelte Official documentation](https://svelte.dev/tutorial/basics)
+- [Svelte for beginners](https://coursetro.com/posts/code/176/Learn-Svelte-3.0---Svelte-Tutorial-for-Beginners)
```
Comment (shahednasser) [inline README.md:509]: As this is a single blog post I think it doesn't qualify it so please remove it.

File: README.md [hunk 2]
```diff
@@ -502,6 +503,16 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [Svelte tutorial by MDN](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_getting_started)
```
Comment (shahednasser) [inline README.md:510]: Please replace the title with the title of the page which is Getting started with Svelte

File: README.md [hunk 3]
```diff
@@ -502,6 +503,16 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [Youtube - The Net Ninja](https://www.youtube.com/playlist?list=PL4cUxeGkcC9hlbrVO_2QFVqVPhlZmz7tO)
```
Comment (shahednasser) [inline README.md:511]: Replace with the title of the Youtube playlist

File: README.md [hunk 4]
```diff
@@ -502,6 +503,16 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [Youtube - Freecodecamp](https://www.youtube.com/watch?v=ujbE0mzX-CU)
```
Comment (shahednasser) [inline README.md:512]: Replace with the title of the video

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @shaji-stark thank you for the contribution. I have added a few comments below please check them out.

**Comment by shaji-stark** [comment]
Sure, I'll edit it right away! Thanks for letting me know ;)

**Comment by shahednasser** [comment]
Thank you for the great work!

**Pull request merged**